components/Themes/tests/unit/MDCTonalPaletteTests.m

```diff
@@ -60,6 +60,17 @@ - (void)testTonalPaletteEncodingDecoding {
   XCTAssertEqual(decodedTonalPalette.mainColorIndex, tonalPalette.mainColorIndex);
   XCTAssertEqual(decodedTonalPalette.darkColorIndex, tonalPalette.darkColorIndex);
   XCTAssertEqual(decodedTonalPalette.lightColorIndex, tonalPalette.lightColorIndex);
+
+  // These tests are commented out because of precision bug on apple side when archiving.
+  // Issue #2931.
+  // TODO: Uncomment these tests when this issue is fixed.
```

Comment:
Super-(SUPER!!!)-nit: I wouldn't bother this time since it will kick off CI again, but consider:
// TODO(#2931): Re-enable tests once Apple's precision bug is fixed
We usually include the issue number in parens after the TODO.

Reply:
I would recommend removing the TODO. I don't think Apple's going to change the behavior - we've just worked around it in other tests.

Reply:
Done.